rest-api/config/options/www.xml

```diff
@@ -62,6 +62,11 @@
       <search:element ns="http://developer.marklogic.com/site/internal" name="_ml_webinar_date"/>
     </search:range>
   </search:constraint>
+  <search:constraint name="EventDate">
+    <search:range type="xs:dateTime" facet="false">
```

Comment:
This requires a range index on end_date, which is not currently configured. Please add a range-element-index to deploy/ml-config.xml (similar to the _ml_webinar_date example).

Reply:
I'll take care of adding this